lib/modules/manager/custom/regex/readme.md

```diff
@@ -1,4 +1,4 @@
-With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
+With `customManagers` using `regex` you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
```

Comment:
Since we are already in the `regex` manager's readme its fine to call it rege manager directly, I think.